src/applications/personalization/profile/components/notification-settings/MissingContactInfoAlertLink.jsx

```diff
@@ -11,30 +13,40 @@ export const MISSING_CONTACT_INFO = {
   MOBILE: 'MOBILE',
 };
 
-const linkMap = {
-  [MISSING_CONTACT_INFO.ALL]: {
-    linkText: 'Update your contact information',
-    linkTarget: getContactInfoDeepLinkURL('phoneNumbers', false),
-  },
-  [MISSING_CONTACT_INFO.EMAIL]: {
-    linkText: 'Add an email address to your profile',
-    linkTarget: getContactInfoDeepLinkURL(FIELD_NAMES.EMAIL, true),
-  },
-  [MISSING_CONTACT_INFO.MOBILE]: {
-    linkText: 'Add a mobile phone number to your profile',
-    linkTarget: getContactInfoDeepLinkURL(FIELD_NAMES.MOBILE_PHONE, true),
-  },
+const makeLinkMap = (missingInfo, flag = false) => {
```

Comment:
Nothing worth holding up a PR over but I wonder if this should be pulled out into the helpers file?

Reply:
Probably a good call. I'll move it to clean this component up a bit. Thanks Jesse